### PR TITLE
yaml: create new 'prod-devel-rcar-virtio.yaml' for VIRTIO configuration

### PR DESCRIPTION
**Pull request by dterletskiy**
Create VIRTIO specific moulin 'prod-devel-rcar-virtio.yaml' configuration. The motivation for this change was that the current YAML file became too bulky, making it difficult to maintain. Therefore, the decision was made to create a separate YAML file for Virtio configuration.

### CODE REVIEW

File: prod-devel-rcar-virtio.yaml [hunk 1]
```diff
@@ -0,0 +1,780 @@
...
+      # is provided as XT_PREBUILT_GSX_DIR variable to components
+      overrides:
+        variables:
+          XT_PREBUILT_GSX_DIR: "${TOPDIR}/../../../prebuilt_gsx"
+        components:
+          domd:
+            builder:
+              conf:
+                - [XT_PREBUILT_GSX_DIR, "%{XT_PREBUILT_GSX_DIR}"]
+          domu:
```
Comment (lorc) [inline prod-devel-rcar-virtio.yaml:751]: Again, no need for DDK in domu

File: prod-devel-rcar-virtio.yaml [hunk 2]
```diff
@@ -0,0 +1,780 @@
...
+
+  ENABLE_ANDROID:
+    desc: "Build Android as a guest VM"
+    "no":
+      default: true
+    "yes":
+      overrides:
+        variables:
+          XT_DOMA_TAG: "doma"
+          XT_USE_DDK1_11: "1"
```
[the remark this answers is not in the document]
Comment (dterletskiy) [reply prod-devel-rcar-virtio.yaml:538]: Do you mean to remove lines 536-538?

File: prod-devel-rcar-virtio.yaml [hunk 3]
```diff
@@ -0,0 +1,780 @@
...
+  DOMZ_DIR: "zephyr"
+  XT_DOMD_DTB_NAME: "%{SOC_FAMILY}-%{MACHINE}-domd.dtb"
+  XT_DOMU_DTB_NAME: "salvator-generic-domu.dtb"
+  XT_DOMA_DTB_NAME: "salvator-generic-doma.dtb"
+  XT_XEN_DTB_NAME: "%{SOC_FAMILY}-%{MACHINE}-xen.dtb"
+  XT_PVR_NUM_OSID: "2"
+  XT_OP_TEE_FLAVOUR: "generic_dt"
+  XT_GENERIC_DOMU_TAG: ""
+  XT_DOMA_TAG: ""
+  XT_DOMA_DDK_KM_PREBUILT_MODULE: ""
```
[the remark this answers is not in the document]
Comment (dterletskiy) [reply prod-devel-rcar-virtio.yaml:20]: Do you mean to remove line 20 only?

[the remark this answers is not in the document]
Comment (lorc) [reply prod-devel-rcar-virtio.yaml:20]: Yes

File: prod-devel-rcar-virtio.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (lorc) [reply prod-devel-rcar-virtio.yaml:538]: Only 538

File: prod-devel-rcar-virtio.yaml (hunk 3 above)
[the remark this answers is not in the document]
Comment (dterletskiy) [reply prod-devel-rcar-virtio.yaml]: Done

File: prod-devel-rcar-virtio.yaml (hunk 2 above)
[the remark this answers is not in the document]
Comment (dterletskiy) [reply prod-devel-rcar-virtio.yaml]: Done

File: prod-devel-rcar-virtio.yaml [hunk 4]
```diff
@@ -0,0 +1,780 @@
...
+            builder:
+              conf:
+                # gles-um-compile.bb still requires python2
+                - [I_SWEAR_TO_MIGRATE_TO_PYTHON3, "yes"]
+
+              layers:
+                - "../meta-python2"
+                - "../meta-clang"
+                - "../meta-xt-rcar/meta-xt-rcar-proprietary"
+          domu:
```
[the remark this answers is not in the document]
Comment (dterletskiy) [reply prod-devel-rcar-virtio.yaml]: Done

File: prod-devel-rcar-virtio.yaml (hunk 1 above)
Comment (dterletskiy) [reply prod-devel-rcar-virtio.yaml]: Done

File: prod-devel-rcar-virtio.yaml [hunk 5]
```diff
@@ -0,0 +1,780 @@
...
+          domd:
+            builder:
+              conf:
+                - [XT_PREBUILT_GSX_DIR, "%{XT_PREBUILT_GSX_DIR}"]
+          domu:
+            builder:
+              conf:
+                - [XT_PREBUILT_GSX_DIR, "%{XT_PREBUILT_GSX_DIR}"]
+
+  ANDROID_PREBUILT_DDK:
```
[the remark this answers is not in the document]
Comment (dterletskiy) [reply prod-devel-rcar-virtio.yaml]: Done

**Review by lorc — APPROVED** [review]
`Acked-by: Volodymyr Babchuk <volodymyr_babchuk@epam.com>`

**Pull request merged**